### PR TITLE
WIP: Python addenda and fixes

### PR DESCRIPTION
**Pull request by dberzano**
This is currently a Work In Progress, I'll remove the `WIP` from the title when
it's ready to be reviewed!

cc our Python expert @mconcas

### CODE REVIEW

File: python/lists.md [hunk 1]
```diff
@@ -68,6 +68,21 @@ specific indices from the list.
...
+```python
+my_function(arg1, arg2, arg3...)
+```
+
+whereas builtins take their arguments _without parentheses_:
+
+```python
+del a[-2]
+```
+{% endcallout %}
```
Comment (alexpearce) [inline python/lists.md]: We should choose the language to be consistent with here. I use “builtins” to refer to anything that’s available in the global namespace in a clean shell, like `dir`, `True`, and `list`. These aren’t “keywords” like `del` and `import`. (I guess I’d say that `print` changed from a keyword to a builtin in Python 3.)

**Review by alexpearce — COMMENTED** [review]
Excellent changes, thank you!

File: python/lists.md (hunk 1 above)
Comment (dberzano) [reply python/lists.md]: You are right, I should not have used the word "builtin" there. Let me change that.

Comment (dberzano) [reply python/lists.md]: OK I have changed it completely now by simply saying that "del is a keyword representing an action".

**Comment by chrisburr** [comment]
Nice clarifications and general tidying! 👍

**Comment by dberzano** [comment]
Uhh this was still a WIP! Nevermind, I'll open a new one :-)

**Comment by chrisburr** [comment]
Ah sorry, no idea how I missed that! 😳

**Pull request merged**